docs/RunningTelegramBotInK8S.md

```diff
@@ -0,0 +1,65 @@
+## Running Telegram bot in K8S
+1. Configure chart as usual
+2. configure persistence, eg:
+```yaml
+telegramBot:
+  persistence:
+    enabled: true
+    storageClass: somestorageclass
+```
+3. configure command:
+```yaml
+command:
+  - python3
+  - -u
+  - telegram_bot.py
+  - --restart-on-init
+```
+Note: `--restart-on-init` is highly recommended to add because in other case some pairs will not have pycryptobots to run after pod restart   
+4. make sure you're allowing enough resources, eg:
+```yaml
+resources:
+ requests:
+   cpu: 500m
+   memory: 1Gi
+ limits:
+   cpu: 2
+   memory: 2.5Gi
+```
+5. It's highly recommend to set `"disablecleanfilesonexit": 1`, in other case if pod gracefully restart you'll lose running crypto-bots
+
```

Comment:
maybe we should provide a complete example or people could ask if these should go into separate yamls?

Reply:
now is more clear?

Reply:
Oh what I meant is providing a full working `values.yaml`. 

like:

Fully working example helm values file:
```
telegramBot:
  persistence:
    enabled: true
    storageClass: somestorageclass
command:
  - python3
  - -u
  - telegram_bot.py
  - --restart-on-init
resources:
 requests:
   cpu: 500m
   memory: 1Gi
 limits:
   cpu: 2
   memory: 2.5Gi
```

Something like that maybe?

Reply:
Although, when I think about it, we can expect people to know how to use helm and kubernetes when they want to go that way I guess.

Reply:
Like this?

Reply:
Yep, thanks, I like it and I think it's more clear. :+1: